src/R7InsightCore/AsyncLogger.cs

```diff
@@ -652,7 +652,7 @@ private void AddLineToQueue(String line, int limit)
                     Queue.TryAdd(trimmedEvent);
                 }
 
-                AddLineToQueue(trimmedEvent.Substring(LOG_LENGTH_LIMIT, trimmedEvent.Length), limit - 1);
+                AddLineToQueue(trimmedEvent.Substring(LOG_LENGTH_LIMIT, trimmedEvent.Length - LOG_LENGTH_LIMIT), limit - 1);
```

Comment:
```suggestion
                AddLineToQueue(trimmedEvent.Substring(LOG_LENGTH_LIMIT), limit - 1);
```
Wouldn't this achieve the same thing?